mongodbatlas/resource_mongodbatlas_project.go

```diff
@@ -8,12 +8,13 @@ import (
 	"net/http"
 	"time"
 
+	"go.mongodb.org/atlas-sdk/v20230201005/admin"
```

Comment:
nit: consistency with declaration of the dependencies

```suggestion
	atlasSDK "go.mongodb.org/atlas-sdk/v20230201005/admin"
```